### ISSUE TITLE
Free-threaded binary builds for 3.13

### ISSUE THREAD

**Issue by mpage** [python/cpython#119243]
# Feature or enhancement

### Proposal:

This is a tracking task for distributing free-threaded binary builds for 3.13

- [x] Ubuntu
- [x] Fedora
- [x] macOS
- [x] Windows
- [x] GH Actions

**Comment by rgommers** [comment]
Windows and macOS can be checked off, they are both supported by the installers for 3.13.0b2 from python.org. Ubuntu is available too, from the `deadsnakes` PPA. I think Fedora is shipping experimental builds themselves. `cibuildwheel` has support too. The `setup-python` GitHub Action not just yet.

https://github.com/Quansight-Labs/free-threaded-compatibility#continuous-integration contains info on CI usage for Python projects.

**Comment by hugovk** [comment]
Yes, Fedora has it in the [python3.13-freethreading package](https://packages.fedoraproject.org/pkgs/python3.13/python3.13-freethreading/).

See also https://dev.to/hugovk/help-us-test-free-threaded-python-without-the-gil-1hgf

**Comment by mwichmann** [comment]
To tie two sets of comments together: if someone structured their project so the rootdir isn't the one with the SConstruct file, then it will remain that way forever (or some variant thereof).  They're not necessarily going to be happy typing `--rootder=path-to-whatever` every single time.  You can't put it in a site file, because site files are located by the.... rootdir - too late.  I think it's better to come up with some sort of permanent marker.
